lib/renderer/webview/webview.h

```diff
@@ -1832,6 +1832,32 @@ struct webview_priv
                     FOS_NOTESTFILECREATE | FOS_NODEREFERENCELINKS |
                     FOS_FORCESHOWHIDDEN | FOS_DEFAULTNOMINIMODE;
       }
+      if (filter[0] != '\0')
+      {
+        int count;
+        int i=0;
+        char* token;
+        char* filter_dup = strdup(filter);
+        for (count=1; filter[count]; filter[count]==',' ? count++ : *filter++);
```

Comment:
Out of interest, why does count start at 1?

Reply:
This counts the number of tokens when splitting the filter string. It is used to size the array used to contain the filter structs required by win32's dialog. So if there's only one filter (no comma) we need an array size of 1.